src/sentry/static/sentry/app/views/releasesV2/detail/overview/chart/discoverChartContainer.tsx

```diff
@@ -23,7 +32,45 @@ type Props = {
   version: string;
 };
 
-class DiscoverChartContainer extends React.Component<Props> {
+type State = {
+  totalEvents: null | number;
+};
+
+class DiscoverChartContainer extends React.Component<Props, State> {
+  state: State = {
+    totalEvents: null,
+  };
+
+  componentDidMount() {
+    this.fetchTotalCount();
+  }
+
+  componentDidUpdate(prevProps: Props) {
+    const {version, organization, selection} = this.props;
+    if (
+      prevProps.version !== version ||
+      !isEqual(prevProps.organization, organization) ||
+      !isEqual(prevProps.selection, selection)
+    ) {
+      this.fetchTotalCount();
+    }
+  }
+
+  async fetchTotalCount() {
+    const {api, organization, location} = this.props;
+
+    try {
+      const totalEvents = await fetchTotalCount(
+        api,
+        organization.slug,
+        this.getEventView().getEventsAPIPayload(location)
+      );
+      this.setState({totalEvents});
+    } catch (err) {
+      Sentry.captureException(err);
```

Comment:
👏

Reply:
This is a bit redundant, Sentry will capture all unhandled rejections. We generally do this if we have our own error handling logic to add.